src/config.rs

```diff
@@ -390,11 +499,36 @@ impl ConfigFile {
             e
         })?;
 
-        let mut result: Self = toml::from_str(&contents).map_err(|e| {
-            tracing::error!("Failed to deserialize {}", config_path.display());
+        let config_file_include_only: ConfigFileIncludeOnly = toml::from_str(&contents).map_err(|e| {
+            tracing::error!("Failed to deserialize the include section of {}", config_path.display());
             e
         })?;
 
+        let mut result: Self = Default::default();
+
+        if let Some(includes) = &config_file_include_only.include {
+            for include in includes.iter().rev() {
+                let include_path = shellexpand::tilde::<&str>(&include.as_ref()).into_owned();
+                let include_path = PathBuf::from(include_path);
+                let include_contents = fs::read_to_string(&include_path).map_err(|e| {
+                    tracing::error!("Unable to read {}", include_path.display());
+                    e
+                })?;
```

Comment:
Wouldn't it be better to simply ignore the file instead of returning an error for the entire read config function?